gradleResources/staticResources/css/ext/input-tags.css

```diff
@@ -6,7 +6,9 @@
 .bootstrap-tagsinput .twitter-typeahead {
 	width: auto;
 	float: none;
-	position: relative !important;
+	position: absolute !important; 
```

Comment:
@stephanrauh i have move the css from bootstrap-tagsinput.css to input-tags.css but i'm not sure if we need relative or absolute